lib/kwtool_web/controllers/api/v1/upload_controller.ex

```diff
@@ -0,0 +1,23 @@
+defmodule KwtoolWeb.Api.V1.UploadController do
+  use KwtoolWeb, :api_controller
+
+  alias Kwtool.Crawler.Keywords
+  alias KwtoolWeb.Api.V1.UploadParams
+
+  def create(conn, params) do
+    user = Guardian.Plug.current_resource(conn)
```

Comment:
We can move this logic to plug to set the `user` in assigns of conn. And in controller, we just get it from conn.
```elixir
def create(conn, params) do
  # `user` available in conn.assigns.user
end
```

Reply:
yup, updated here https://github.com/byhbt/kwtool/pull/46/commits/80443a6a693075a9b6a818ef06a36a1e06ac7611